### DISCUSSION TITLE
The status of wallet joint cultivation has been pending confirmation for several days now

### DISCUSSION THREAD

**Discussion by GitCHanGY** [Chia-Network/chia-blockchain#17164] [Support]
Joint farming has been earning points and the status has been pending confirmation, and the mining pool shows that I have exited the mining pool
![微信图片_20231226140827](https://github.com/Chia-Network/chia-blockchain/assets/39291464/ba790b38-7148-4744-9cff-0368d62eacc7)
![微信图片_20231226140822](https://github.com/Chia-Network/chia-blockchain/assets/39291464/7438b295-22a7-47aa-864d-12e271b046eb)

**Comment by GitCHanGY** [comment]
After deleting an unconfirmed transaction, the status will be displayed as invalid
![1](https://github.com/Chia-Network/chia-blockchain/assets/39291464/bf873976-f380-45ce-8a43-68f4dfc2c093)

**Comment by BrandtH22** [comment]
Hello @GitCHanGY , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

Were you attempting to switch pools or start self pooling prior to this error arising?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
